TESTS/netsocket/tcp/main.cpp

```diff
@@ -72,7 +73,17 @@ static void _ifup()
     NetworkInterface *net = NetworkInterface::get_default_instance();
     nsapi_error_t err = net->connect();
     TEST_ASSERT_EQUAL(NSAPI_ERROR_OK, err);
-    printf("MBED: TCPClient IP address is '%s'\n", net->get_ip_address() ? net->get_ip_address() : "null");
+    const char *address = net->get_ip_address();
+
+    printf("MBED: TCPClient IP address is '%s'\n", address ? address : "null");
+#if MBED_CONF_TARGET_NETWORK_DEFAULT_INTERFACE_TYPE == MESH
+    uint8_t addr_hex[16];
+    stoip6(address, strlen(address), addr_hex);
+    if ((addr_hex[0] & 0xe0) != 0x20) {
+        printf("Adress is not globlal  Please check Border Router  TEST SKIP \n");
```

Comment:
Please fix typos adress -> Address and globlal -> global (+ 2 other files where copy pasted same mistake)

Reply:
Changed